corehq/apps/geospatial/utils.py

```diff
@@ -33,6 +33,12 @@ def get_geo_user_property(domain):
     return config.user_location_property_name
 
 
+def get_celery_task_tracker(domain, base_key):
```

Comment:
nit: make name specific to the task that it is tracking, `get_celery_task_tracker` seems too generic

Reply:
@AmitPhulera If we make this function's name specific to the task, then we'll have to create another function for getting another task's tracker in the future, so in my mind this is OK since it does describe what the function does (at least, that's how I think about naming functions). Feel free to disagree. I'll leave the final call to Zandre.

I would suggest updating the `base_key` argument name, since it's not immediately obvious what value that will hold. Maybe think something along the lines of "task_name" or "task_slug"...something to say it's going to identify the task running on the domain.

Reply:
I agree with @Charl1996 here, and this was my sentiment when creating this function. My thinking is that the `CeleryTaskTracker` class has potential future use outside the scope of this specific task. I kept the naming generic here so that it would make sense in the future should we ever decide to extend the usage of this class. But I'm also happy to rename if @AmitPhulera feels strongly about this. If/When we ever need to extend this in the future, we can always reassess renaming this helper function.

> I would suggest updating the base_key argument name

I'm fine with updating this, I'll change it to `task_slug`

Reply:
I didn't have very strong opinion about it. It's just that we have similar implementations with slight variations which do almost the same thing. I was not sure it this util something like that in future. I am fine with your judgement.